### ISSUE TITLE
I'm not getting udates anymore

### ISSUE THREAD

**Issue by canoben** [Dielee/volvo2mqtt#256]
**Describe the bug**
Since update to 1.10.5 entities are no longer shown as unavailable, but they are showing an old value. When I updates yesterday to version 10.4 the integration updated all the available values. This morning however (after a full night of recharging my car) battery status is still at 12% (instead of 90%).

**Version info**:<br>
v.1.10.5 in VM on Synology NAS (all the latest updates of HA installed)

**Logs**:<br>
There are no log entries relating to volvo2mqtt

**Comment by KristianAndersson** [comment]
Still have this issue and does the app show the correct info?

I had this issue, but the Volvo app also didn't have updated info, which in my case didn't point to a bug with Volvo2mqtt.

I contacted Volvo support and was sent through a car "data calibration" / reset procedure. After this the app and Volvo2mqtt had updated info. If this actually was the fix, or if something was fixed on Volvos side while doing the reset I don't know.
